packages/launcher/lib/linux/index.ts

```diff
@@ -3,8 +3,33 @@ import type { FoundBrowser, Browser, PathData } from '../types'
 import { notInstalledErr } from '../errors'
 import { utils } from '../utils'
 import os from 'os'
+import { promises as fs } from 'fs'
 import path from 'path'
 import Bluebird from 'bluebird'
+import which from 'which'
+
+async function isFirefoxSnap (binary: string): Promise<boolean> {
+  try {
+    return await Bluebird.resolve((async () => {
```

Comment:
I'd like to avoid Bluebird promises in newly written code when possible. Especially since we're already inside an `async` function, do we need `Bluebird.resolve` here?

Reply:
I'm using Bluebird.resolve to get access to `.timeout`. I could `Promise.race/new Promise/setTimeout` but I figure that unless we are planning on moving all the code in the code base to not use Bluebird, might as well use the utilities we have. I also generally agree that BB should be avoided in new code.

Reply:
That is something I would like to do at some point, remove bluebird from our existing code. But adding timeout utility functions is beyond the scope of this PR, fair enough.